insights/parsers/rhui_release.py

```diff
@@ -55,10 +55,14 @@ def parse_content(self, content):
         if len(content) == 1:
             rhel_version = content[0].strip()
             line_splits = rhel_version.split('.')
-            if len(line_splits) == 2 and line_splits[0].isdigit() and line_splits[-1].isdigit():
+            if len(line_splits) == 2:
+                if line_splits[0].isdigit() and line_splits[-1].isdigit():
+                    self.set = rhel_version
+                    self.major = int(line_splits[0])
+                    self.minor = int(line_splits[-1])
+                    return
+            else:
                 self.set = rhel_version
```

Comment:
@JoySnow Yes, I checked it too, but I thought if the version is like "7Server" or "8", the major version is useless since it's wrong, we always check the minor version to confirm the right release is set. But I'm good to keep it as the others in case we may use it in the future.

Reply:
@JoySnow Please help to review again, thanks.